tests/utils.js

```diff
@@ -157,6 +157,40 @@ describe('Utils', () => {
       chai.expect(utils.hideAbsolutePathsInObject(object)).to.deep.equal(object);
     });
 
+    it('should return a plain object when given an error object', function() {
+      const error = new Error('Hello, World!');
+      const hiddenPaths = utils.hideAbsolutePathsInObject(error);
+
+      chai.expect(hiddenPaths).to.deep.equal({
+        message: 'Hello, World!',
+        stack: utils.removeContainedPaths(error.stack)
```

Comment:
Yuck, would it be possible to do this without making function calls inside the `expect to equal` section?